.github/workflows/CI.yml

```diff
@@ -1,8 +1,10 @@
 name: CI
 
 on:
-  - pull_request
-  - push
+  pull_request:
+  push:
+    branches:
+      - master
```

Comment:
It can be valuable to run CI on a WIP branch, so I'd personally like the `push` trigger to stay active for _all branches_, not just `master`.

Reply:
So, as long as a PR is created it'll still run the CI on pushes. It just won't run _both_ whenever a push happens